src/test/scala/com/cognite/spark/datasource/EventsRelationTest.scala

```diff
@@ -22,6 +22,24 @@ class EventsRelationTest extends FlatSpec with Matchers with SparkTest {
     assert(res.length == 1000)
   }
 
+  it should "apply a single pushdown filter" taggedAs WriteTest in {
+    val df = spark.read.format("com.cognite.spark.datasource")
+      .option("apiKey", writeApiKey)
+      .option("type", "events")
+      .load()
+      .where(s"type = 'future'")
+    assert(df.count == 1000)
+  }
+
+  it should "apply multiple pushdown filters" taggedAs WriteTest in {
+    val df = spark.read.format("com.cognite.spark.datasource")
+      .option("apiKey", writeApiKey)
+      .option("type", "events")
+      .load()
+      .where(s"type = '***' and subtype = 'Valhall'")
```

Comment:
Add a test for "or" as well, and a couple of tests doing multiple types and subtypes.